app/models/identity_tijuana/user.rb

```diff
@@ -8,14 +8,14 @@ class User < ApplicationRecord
 
     scope :updated_users, -> (last_updated_at) {
       includes(:postcode)
-      .where('users.updated_at >= ?', last_updated_at)
+      .where('users.updated_at > ?', last_updated_at)
       .order('users.updated_at')
       .limit(Settings.tijuana.pull_batch_amount)
     }
 
     scope :updated_users_all, -> (last_updated_at) {
       includes(:postcode)
-      .where('users.updated_at >= ?', last_updated_at)
+      where('users.updated_at > ?', last_updated_at)
```

Comment:
Is the removed `.` here a problem?

Reply:
No, it's needed because of the insertion of the includes() method beforehand. There's message chaining going on here, so the first call doesn't need a . prefix, but all subsequent ones do.

Reply:
This looks like a mistake made when I was allocating multiple changes to different commits. The preceding includes(:postcode) line was removed in a subsequent commit, so from that point on we're good. Is it worth trying to fix this up? I don't think it would raise an error invoking the "where" without message chaining, but I suspect the includes() method might have no effect.

Reply:
Executive decision made that it's not worth the trouble of going back and fixing the individual commits given that the end result is correct.